src/torch_onnx_models/components/_rotary_embedding.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+import math
+
+import torch
+from torch import nn
+
+from torch_onnx_models import _configs
+from torch_onnx_models.components._rotary_embedding_utils import get_rotary_pos_emb
+
+
+class Llama3Rope(nn.Module):
```

Comment:
I thought there was like a few types: default linear etc. Or was it much more complex?